CRITICAL - 2024-09-04 00:00:38 --> ErrorException: Undefined variable $datosbd
[Method: GET, Route: ver_cliente]
in APPPATH\Views\Clientes.php on line 28.
 1 APPPATH\Views\Clientes.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datosbd', 'C:\\xampp\\htdocs\\parcial1_jt\\app\\Views\\Clientes.php', 28)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\parcial1_jt\\app\\Views\\Clientes.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('Clientes', [], true)
 5 APPPATH\Controllers\ControladorCliente.php(15): view('Clientes', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:00:44 --> ErrorException: Undefined variable $datosbd
[Method: GET, Route: ver_cliente]
in APPPATH\Views\Clientes.php on line 28.
 1 APPPATH\Views\Clientes.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datosbd', 'C:\\xampp\\htdocs\\parcial1_jt\\app\\Views\\Clientes.php', 28)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\parcial1_jt\\app\\Views\\Clientes.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('Clientes', [], true)
 5 APPPATH\Controllers\ControladorCliente.php(15): view('Clientes', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:36 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:36 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:37 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:37 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:37 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:38 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:41 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:41 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:42 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:27:44 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:18 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:19 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:19 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:19 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:19 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:19 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:23 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:25 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:25 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:25 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 00:28:25 --> ParseError: syntax error, unexpected end of file
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:34:46 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:34:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:34:59 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:34:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:40:32 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:40:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:40:38 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:40:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:40:38 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:40:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:40:39 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:40:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:40:39 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:40:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:41:26 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:41:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:43:15 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:43:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 06:49:43 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'bd_dise03', 3306, '', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bd_dise03' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\parcial1_jt\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
#7 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-09-04 06:49:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bd_dise03'
[Method: GET, Route: ver_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ControladorCliente.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->verCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:01:27 --> ArgumentCountError: Too few arguments to function App\Controllers\ControladorCliente::eliminarCliente(), 0 passed in C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: GET, Route: eliminar_cliente]
in APPPATH\Controllers\ControladorCliente.php on line 52.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->eliminarCliente()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 08:09:50 --> mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_jt\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#10 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-04 08:09:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
[Method: POST, Route: guardar_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:09:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Lopez\', \'Guatemala\', \'789456123\', \'jim@lopez.com\', \'45678951\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Lopez\', \'Guatemala\', \'789456123\', \'jim@lopez.com\', \'45678951\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:09:50 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Lopez\', \'Guatemala\', \'789456123\', \'jim@lopez.com\', \'45678951\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Lopez\', \'Guatemala\', \'789456123\', \'jim@lopez.com\', \'45678951\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Lopez\', \'Guatemala\', \'789456123\', \'jim@lopez.com\', \'45678951\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 08:14:57 --> mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_jt\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#10 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-04 08:14:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
[Method: POST, Route: guardar_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:14:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:14:57 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 08:15:05 --> mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_jt\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#10 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-04 08:15:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
[Method: POST, Route: guardar_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:15:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:15:05 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Peter\', \'Anguila\', \'Guatemala\', \'78454646\', \'peter@anguila.com\', \'55447788\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-09-04 08:17:56 --> mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_jt\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#10 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-04 08:17:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
[Method: POST, Route: guardar_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:17:56 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'88888888\', \'bkrjygo@gmail.com\', \'49639648\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'88888888\', \'bkrjygo@gmail.com\', \'49639648\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:17:56 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'88888888\', \'bkrjygo@gmail.com\', \'49639648\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'88888888\', \'bkrjygo@gmail.com\', \'49639648\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'88888888\', \'bkrjygo@gmail.com\', \'49639648\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 08:25:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-09-04 08:25:49 --> mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY' in C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\parcial1_jt\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `cl...', 0)
#1 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cl...')
#2 C:\xampp\htdocs\parcial1_jt\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cl...')
#3 C:\xampp\htdocs\parcial1_jt\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cl...', Array, false)
#4 C:\xampp\htdocs\parcial1_jt\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\parcial1_jt\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\parcial1_jt\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\parcial1_jt\app\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
#9 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
#10 C:\xampp\htdocs\parcial1_jt\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\parcial1_jt\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\parcial1_jt\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\parcial1_jt\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-09-04 08:25:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
[Method: POST, Route: guardar_cliente]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:25:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'4564564654\', \'bkrjygo@gmail.com\', \'49639648\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'4564564654\', \'bkrjygo@gmail.com\', \'49639648\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:25:49 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'4564564654\', \'bkrjygo@gmail.com\', \'49639648\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'4564564654\', \'bkrjygo@gmail.com\', \'49639648\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (\'Jimy\', \'Tepéu\', \'2da calle\', \'4564564654\', \'bkrjygo@gmail.com\', \'49639648\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `clientes` (`nombre`, `apellido`, `direccion`, `nit`, `correo_electronico`, `telefono_movil`) VALUES (:nombre:, :apellido:, :direccion:, :nit:, :correo_electronico:, :telefono_movil:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ControladorCliente.php(45): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->guardarCliente()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 08:25:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:42:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:42:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:42:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:42:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:50:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:50:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:50:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:50:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 08:51:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-09-04 08:55:39 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: ver_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:39 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: ver_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:40 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: ver_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:40 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: ver_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:40 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: crear_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:42 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: ver_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:42 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: /]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-04 08:55:43 --> Error: Call to undefined method CodeIgniter\Router\RouteCollection::name()
[Method: GET, Route: crear_cliente]
in APPPATH\Config\Routes.php on line 20.
 1 SYSTEMPATH\Router\RouteCollection.php(340): require()
 2 SYSTEMPATH\CodeIgniter.php(823): CodeIgniter\Router\RouteCollection->loadRoutes()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 09:07:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-09-04 09:07:18 --> Error: Call to undefined function form_open()
[Method: GET, Route: editar_cliente/2]
in APPPATH\Views\editar_cliente.php on line 13.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('editar_cliente', [], true)
 4 APPPATH\Controllers\ControladorCliente.php(74): view('editar_cliente', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->editarCliente('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 09:07:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:12:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:12:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:12:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:12:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-09-04 09:12:51 --> Error: Call to undefined function form_open()
[Method: GET, Route: editar_cliente/2]
in APPPATH\Views\editar_cliente.php on line 13.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('editar_cliente', [], true)
 4 APPPATH\Controllers\ControladorCliente.php(74): view('editar_cliente', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->editarCliente('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 09:17:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-09-04 09:17:56 --> Error: Call to undefined function form_open()
[Method: GET, Route: editar_cliente/2]
in APPPATH\Views\editar_cliente.php on line 13.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('editar_cliente', [], true)
 4 APPPATH\Controllers\ControladorCliente.php(74): view('editar_cliente', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->editarCliente('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 09:17:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-09-04 09:17:59 --> Error: Call to undefined function form_open()
[Method: GET, Route: editar_cliente/2]
in APPPATH\Views\editar_cliente.php on line 13.
 1 SYSTEMPATH\View\View.php(234): include()
 2 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('editar_cliente', [], true)
 4 APPPATH\Controllers\ControladorCliente.php(74): view('editar_cliente', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ControladorCliente->editarCliente('2')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControladorCliente))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-04 09:18:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:20:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:20:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:20:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:20:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:20:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:20:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:21:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:21:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:29:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:47:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:48:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:48:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:48:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:50:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:50:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:50:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:50:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:50:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 09:59:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:09:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:13:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:14:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:14:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:15:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:16:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:16:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:16:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:16:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-04 10:16:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
